build-utils/language-go/src/test/resources/plc4go/pom.xml

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
was just wondering if we find a way to keep these in sync with the other poms or if we use tool native builds here.

Reply:
mostly this is for me a discussion reminder. (pom is pretty big, also plc4go received some updates in the meantime, artifactid should be something like example.com:example e.g.)

Reply:
I have added a reference to the parent pom of build-utils for the integration tests. It would be nice to be able to keep it up to date with the plc4go pom, but not sure there is a way to do it.

At least for the integration tests I don't think there is much value a native tool as the Go build-utils is just a java project that happens to produce Go source code.